### PR TITLE
sync some more types with PhpStorm stubs

### PR DESCRIPTION
**Pull request by voku**
⇾ found via: https://github.com/voku/php-doc-fixer#command-for-analysing-phpstan-stubs

---
This change is [Reviewable](https://reviewable.io/reviews/phpstan/phpstan-src/387)

### CODE REVIEW

File: resources/functionMap.php [hunk 1]
```diff
@@ -377,7 +377,7 @@
 'asinh' => ['float', 'number'=>'float'],
 'asort' => ['bool', '&rw_array_arg'=>'array', 'sort_flags='=>'int'],
 'assert' => ['bool', 'assertion'=>'string|bool', 'description='=>'string|Throwable|null'],
-'assert_options' => ['mixed', 'what'=>'int', 'value='=>'mixed'],
+'assert_options' => ['object|array|string|int|null', 'what'=>'int', 'value='=>'mixed'],
```
Comment (ondrejmirtes) [inline resources/functionMap.php:380]: There's definitely missing `false` here, see the docs https://www.php.net/manual/en/function.assert-options.php

Comment (ondrejmirtes) [reply resources/functionMap.php:380]: But at this point, the `mixed` should stay as basically any type can be returned...

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

File: resources/functionMap.php (hunk 1 above)
Comment (voku) [reply resources/functionMap.php]: Changes to ```assert_options``` have been rolled back.

**Comment by ondrejmirtes** [comment]
Thank you!

**Pull request merged**